pkg/sidecars/lb_sidecar/handlers.go

```diff
@@ -0,0 +1,160 @@
+package sidecarserv
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/inspr/inspr/cmd/insprd/memory/tree"
+	"github.com/inspr/inspr/pkg/environment"
+	"github.com/inspr/inspr/pkg/ierrors"
+	metautils "github.com/inspr/inspr/pkg/meta/utils"
+	"github.com/inspr/inspr/pkg/rest"
+	"go.uber.org/zap"
+)
+
+var logger *zap.Logger
+
+func init() {
+	logger = zap.NewNop()
+
+	// logger, _ = zap.NewProduction(zap.Fields(zap.String("section", "loadbalencer-sidecar")))
+}
+
+// writeMessageHandler handles requests sent to the write message server
+func (s *Server) writeMessageHandler() rest.Handler {
+	return func(w http.ResponseWriter, r *http.Request) {
+		logger.Info("handling message write")
+
+		channel := strings.TrimPrefix(r.URL.Path, "/")
+
+		if !environment.OutputChannnelList().Contains(channel) {
+			logger.Error("channel " + channel + " not found in output channel list")
+			insprError := ierrors.NewError().
+				BadRequest().
+				Message("channel '%s' not found", channel)
+
+			rest.ERROR(w, insprError.Build())
+			return
+		}
+
+		channelBroker, err := getChannelBroker(channel)
+		if err != nil {
+			logger.Error("unable to get channel broker",
+				zap.String("channel", channel),
+				zap.Any("error", err))
+			rest.ERROR(w, err)
+			return
+		}
+
+		sidecarWritePort := os.Getenv("INSPR_SIDECAR_" + strings.ToUpper(channelBroker) + "_WRITE_PORT")
+		if sidecarWritePort == "" {
+			logger.Error("unable to get broker " + channelBroker + " port")
+			insprError := ierrors.NewError().
+				NotFound().
+				Message("[ENV VAR] INSPR_SIDECAR_%s_WRITE_PORT not found", channelBroker)
+
+			rest.ERROR(w, insprError.Build())
+			return
+		}
+
+		reqAddress := fmt.Sprintf("http://localhost:%v/%v", sidecarWritePort, channel)
```

Comment:
Use the base address as an env var aswell, just in case we do different types of sidecars in the future.